metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -641,42 +572,23 @@ def parse_view_lineage(
         return list(parsed_tables)
 
     def _compute_bigquery_lineage(self, project_id: str) -> Dict[str, Set[LineageEdge]]:
-        lineage_extractor: BigqueryLineageExtractor = BigqueryLineageExtractor(
-            config=self.config, report=self.report
-        )
```

Comment:
No clue why we were doing this... was in this code from the very first bigquery lineage PR. The only instance variables are `self.config` and `self.report`, which are passed thru anyway, so I think this should be safe to remove